lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1866,6 +1886,22 @@ def api_client
       @client
     end
 
+    def use_secure_channel
+      @logging_api_url.start_with?('https')
+    end
+
+    def grpc_channel_name
```

Comment:
Probably makes sense to inline this method into `init_api_client` as well, allowing you to use `uri.scheme` instead of the `starts_with` string test.

Reply:
Changed.